msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -75,7 +75,16 @@ For `serviceUrl`, use the value from an incoming activity triggering the flow or
 * GCC High: `https://smba.infra.gov.teams.microsoft.us/teams`
 * DoD: `https://smba.infra.dod.teams.microsoft.us/teams`
 
-For a code sample, see the call `CreateConversationAsync` in the [**sample**](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs).
+> [!WARNING]
+>
+> * Avoid hardcoding URLs. Use `serviceUrl` from the incoming activity or conversation reference from the bot. If it's unavailable, use global URLs based on region and cloud.
+>
+> * These URLs are for proactive messages only. For any reply to messages, use the `serviceURL` from the incoming request. For more information, see [Activity.ServiceUrl property](/dotnet/api/microsoft.bot.schema.activity.serviceurl?view=botbuilder-dotnet-stable&preserve-view=true).
```

Comment:
Let's put this "These URLs are for proactive messages only." at appropriate place in first point.